<launch>

	<node pkg="course_agv_slam_task" type="particle_filter" name="particle_filter" output="screen">
		
		<!--from a rosbag-->
        <param name="init_x" value="0"/>
        <param name="init_y" value="0"/>
        <param name="init_theta" value="0"/>

        <param name="init_rand_xy" value="5"/>
        <param name="init_rand_theta" value="6.28"/>
        <param name="obstacle_std" value="0.4"/>
        <param name="Neff" value="0.02"/>
        <param name="gen_var" value="0.01"/>
        <param name="gen_exp" value="0.15"/>
        <param name="laser_max_samples" value="20"/>
	</node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find course_agv_slam_task)/launch/course_agv.rviz" />
</launch>
